file_paths.md

```diff
@@ -0,0 +1,108 @@
+        hiera.yaml                        # hiera_config
+        hieradata                         # n/a
+        modules                           # (user modulepath used in basemodulepath)
+
+    /etc/puppetlabs/mcollective
+        client.cfg
+        facts.yaml
+        server.cfg
+
+    /etc/puppetlabs/puppet                # confdir                      /etc/puppet
+        auth.conf                         # rest_authconfig
+        autosign.conf                     # autosign
+        binder_config.yaml                # binder_config
+        csr_attributes.yaml               # csr_attributes
+        custom_trusted_oid_mapping.yaml   # trusted_oid_mapping_file
+        device.conf                       # deviceconfig
+        fileserver.conf                   # fileserverconfig
+        puppet.conf                       # config
+        routes.yaml                       # route_file
+        ssl                               # ssldir                       /etc/puppet/ssl
+
+    /opt/puppetlabs/bin                   # symlink targets of puppet related binaries
+        cfacter@
+        facter@
+        hiera@
+        mco@
+        mcollectived@
+        puppet@
+
+    /opt/puppetlabs/mcollective/
+        plugins
+
+    /opt/puppetlabs/puppet                # ruby-puppet root
+        bin
+            cfacter
+            facter
+            gem
+            hiera
+            mco
+            mcollectived
+            openssl
+            puppet
+            ruby
+        cache                             # vardir                       /var/lib/puppet
+            bucket                        # bucketdir
+            client_yaml                   # clientyamldir
+            client_data                   # client_datadir
+            clientbucket                  # clientbucketdir
+            devicedir                     # devices
+            facts.d                       # pluginfactdest
+            lib                           # libdir
+            facts                         # (used in factpath)
+            puppet-module                 # module_working_dir
+            reports                       # reportdir
+            server_datadir                # server_data
+            state                         # statedir
+            yaml                          # yamldir
+        include
+        lib
+        modules                           # (system modulepath)          /usr/share/puppet/modules
+        share
+            augeas
+            man
+            vim
+        ssl
+
+    /var/log/puppetlabs                   # logdir                       /var/lib/puppet/log
+        puppet.log
+        mcollective.log
+        mcollectived.log
+
+
+    /var/run/puppetlabs                   # rundir                       /var/lib/puppet/run
+        agent.pid
+        mcollective.pid
+        mcollectived.pid
+
+
+puppetdb
+--------
+
+    /etc/puppetlabs/puppetdb
+        puppetdb.conf
```

Comment:
Having said that, I have no idea what we are trying to express here, whether we are just throwing out all of PuppetDB's configuration, or only the bits that Puppet cares about (like the terminus configuration). Either way this is wrong.